e2e/cypress/test_module/cypress/features/home-page/home-page-general.feature

```diff
@@ -29,6 +36,16 @@ Feature: | Guest Home Page |
     When I click on Shop Now button
     Then I should be redirected to Products Page
 
+  Scenario: Call-to-action section - redirect to category page
+    When I look through Call-to-action section and click on '<category>' banner button
+    Then I should be redirected to '<category>' category page
+
+    Examples:
+      | category |
+      | mobile   |
+      | computer |
```

Comment:
Are you sure it works correctly? I guess it should be `Scenario Outline` instead of just `Scenario`

Reply:
fixed